src/pages/ProductsPage.tsx

```diff
@@ -21,6 +21,12 @@ export const ProductsPage: React.FC = () => {
   const paths: string[] = pathname.split('/');
   const currentPath = paths[1];
 
+  const pageNumbers: number[] = [];
+
+  for (let i = 1; i <= Math.ceil(allProducts.length / numberProductsOnPage); i++) {
```

Comment:
Why you didn't use method forEach instead for ?

Reply:
maybe you can tell me how to use it here?)